PublStatus.md

```diff
@@ -9,51 +9,50 @@ layout: default
 
 ### Active documents
 
-| Document | Latest official release | Editors’ Draft | Repository |
-|----------|:-----------------------:|:--------------:|:----------:|
-| Web Publications—Manifest and Structure       | [wd](https://www.w3.org/TR/wpub/) | [ed](https://w3c.github.io/wpub/)              | [wpub](https://github.com/w3c/wpub/) |
-| Lightweight Packaging of Web Publications† | (none yet) | ([ed](https://w3c.github.io/pwpub/spec/ocf-lite.html))         | [pwpub](https://github.com/w3c/pwpub/)                |
-| AudioBooks WP Profile | [wd](https://www.w3.org/TR/audiobooks/) | [ed](https://w3c.github.io/audiobooks/)         | [audiobooks](https://github.com/w3c/audiobooks/)                |
-| Web Publications Use Cases and Requirements‡ | [note](https://www.w3.org/TR/pwp-ucr/) | [ed](https://w3c.github.io/dpub-pwp-ucr/)| [dpub-pwp-ucr/](https://github.com/w3c/dpub-pwp-ucr//)                |
+| Document | Latest official release | Editors’ Draft | Repository | Note |
+|----------|:-----------------------:|:--------------:|:----------:|:---:|
+| Publication Manifests | (none yet) | [ed](https://w3c.github.io/pub-manifest/)         | [pub-manifest](https://github.com/w3c/pub-manifest/)    | Rec-track |
+| AudioBooks WP Profile | [wd](https://www.w3.org/TR/audiobooks/) | [ed](https://w3c.github.io/audiobooks/)         | [audiobooks](https://github.com/w3c/audiobooks/)                | Rec-track |
+| Lightweight Packaging of Web Publications | (none yet) | [ed](https://w3c.github.io/lpf/)         | [lpf](https://github.com/w3c/lpf/) | Not rec-track
```

Comment:
Same question about appropriateness of "of Web Publications" in this title.

Reply:
Again, true, and have changed it